ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_TENANTS_NETWORKS/Tenant_A.yml

```
@@ -255,6 +255,14 @@ tenant_a:
             enabled: True
             description: "Single description"
             descriptions: ["DC1-BL1A descriptions preferred over single description", "DC1-BL1B descriptions preferred over single description"]
+          - interfaces: [Ethernet12, Ethernet13, Ethernet12, Ethernet13]
```

Maybe we can add a test with subinterface to make sure we never break it here
```suggestion
          - interfaces: [Ethernet12, Ethernet13.10, Ethernet12, Ethernet13.10]
```

---

Added.